### PR TITLE
Couldinary

### PR DESCRIPTION
**Pull request by ajkatom**
### What this PR does (required):
created controllers for and set up cloudinary
tested with postman but cannot be tested in the app because is depends on profile FE.

### CODE REVIEW

File: server/models/Profile.js [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
     type: Date,
-    default: null
+    default: null,
   },
   photo: {
     type: String,
-    default: "",
+    default: '',
+  },
+  cloudinary_id: {
```
Comment (moffatethan) [inline server/models/Profile.js:37]: Super close on this one - could we just make it camelCase. Everything else looks great! Make sure to change the other files to use `cloudinaryId`

File: server/controllers/profile.js [hunk 2]
```diff
@@ -1,8 +1,84 @@
...
+  params: {
+    folder: 'images',
+    public_id: (req, file) => {
+      file.originalname + file.size;
+    },
+  },
+});
+const upload = multer({ storage: storage }).single('File');
 
+// @route POST /profile//uploadPicture
```
Comment (moffatethan) [inline server/controllers/profile.js:18]:
```suggestion
// @route POST /profile/uploadPicture
```

**Review by moffatethan — CHANGES_REQUESTED** [review]
Super close on this one - great work with the cloudinary implementation and attaching profile id to the request object. 

Just two minor comments there then I'll approve.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Please fix up the merge conflicts then I'll approve 👍🏻

**Comment by ajkatom** [comment]
> 

done

**Pull request merged**